src/config.c

```diff
@@ -2633,6 +2634,7 @@ standardConfig configs[] = {
     createIntConfig("min-replicas-to-write", "min-slaves-to-write", MODIFIABLE_CONFIG, 0, INT_MAX, server.repl_min_slaves_to_write, 0, INTEGER_CONFIG, NULL, updateGoodSlaves),
     createIntConfig("min-replicas-max-lag", "min-slaves-max-lag", MODIFIABLE_CONFIG, 0, INT_MAX, server.repl_min_slaves_max_lag, 10, INTEGER_CONFIG, NULL, updateGoodSlaves),
     createIntConfig("watchdog-period", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.watchdog_period, 0, INTEGER_CONFIG, NULL, updateWatchdogPeriod),
+    createIntConfig("aof-child-rewrite-delay", NULL, MODIFIABLE_CONFIG, INT_MIN, INT_MAX, server.aof_child_rewrite_delay, 0, INTEGER_CONFIG, NULL, NULL),
```

Comment:
`INT_MIN` doesn't seem right.
This one is just for testing, but worth mentioning in the top comment anyway.
Thing is that some reviewers (mainly the core team), may not bother to read the entire source code, and will approve this PR based on the detailed top comment.